### PR TITLE
LPD-31478 add RTL for languages

### PR DESCRIPTION
**Pull request by algobob**
Resent from #670

### CODE REVIEW

**Comment by algobob** [comment]
ci:test:sf

**Comment by algobob** [comment]
ci:test:sf

**Comment by algobob** [comment]
ci:test:relevant

File: modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json [hunk 1]
```diff
@@ -389,6 +389,12 @@
 								},
 								"mainURL": {
 									"type": "string"
+								},
+								"clayRTLURL": {
+									"type": "string"
+								},
+								"mainRTLURL": {
+									"type": "string"
```
Comment (drewbrokke) [inline modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json:397]: Brian will want these keys sorted alphabetically.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json (hunk 1 above)
Comment (algobob) [reply modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json:397]: done

**Review by drewbrokke — APPROVED** [review]
LGTM, thanks @algobob!

**Comment by algobob** [comment]
ci:forward

**Pull request closed without merging**